lib/game.rb

```diff
@@ -10,14 +10,13 @@ class Game
               :venue,
               :venue_link
 
-  def initialize(row, parent)
-    @parent = parent
+  def initialize(row)
```

Comment:
This was a great catch, if a particular class is never going to ask its parent for something there is no sense in passing it the parent